web/src/beta/components/StorytellingPageSectionItem/index.tsx

```diff
@@ -0,0 +1,74 @@
+import styled from "@emotion/styled";
+import { FC } from "react";
+
+import Icon from "../Icon";
+import Text from "../Text";
+
+type Props = {
+  icon: string;
+  title: string;
+  index: number;
+  active?: boolean;
+  onAction?: () => void;
+  onClick?: () => void;
+};
+
+const StorytellingPageSectionItem: FC<Props> = ({
+  icon,
+  title,
+  index,
+  active,
+  onAction,
+  onClick,
+}) => {
+  return (
+    <HorizontalBox>
+      <VerticalBox>
+        <Text size={"m"} color="#c7c5c5">
+          {index}
+        </Text>
+        <Icon icon={icon} />
+      </VerticalBox>
+      <TitleArea active={active}>
+        <Text onClick={onClick} size={"s"} color="#ffffff">
+          {title}
+        </Text>
+        <Icon icon={"actionbutton"} onClick={onAction} />
+      </TitleArea>
+    </HorizontalBox>
+  );
+};
+
+const HorizontalBox = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
```

Comment:
As far as I remember, this two properties are default behaviour. So don't need it.